packages/velo-external-db-core/lib/service/schema.spec.js

```diff
@@ -26,7 +27,7 @@ describe('Schema Service', () => {
         driver.givenSupportedOperations(ctx.schemaOperations)
 
         const actual = await env.schemaService.find(ctx.dbs.map(db => db.id))
-        expect( actual ).toEqual({ schemas: ctx.dbs.map( db => asWixSchema(db, ctx.schemaOperations) ) })
+        expect( actual ).toEqual({ schemas: schemasList(ctx.dbs, ctx.schemaOperations) })
```

Comment:
great work on the matcher, but let's try to work on the name 
```haveSchemaFor(db, opts)```
would be more readable IMHO